Automattic/newspack-plugin
# feat(ras-acc): add notice styles to newspack ui

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207720049982626/1207546771559780/f

This PR adds notice styles to newspack ui along with a notices section to the ui demo:
![Screenshot 2024-07-10 at 12 57 53](https://github.com/Automattic/newspack-plugin/assets/17905991/8435fb71-4e62-41fd-bd4d-161670fc1e1b)

### How to test the changes in this Pull Request:

1. With this branch checked out access the ui demo by adding the following url param to your local dev: `?ui-demo` (you must be logged in as admin)
2. Scroll down to the new notices section and confirm it appears as pictured above

### Other information:
